tools/makeboards.py

```diff
@@ -260,6 +261,8 @@ def BuildHeader(name, chip, chaintuple, chipoptions, vendor_name, product_name,
     print('%s.build.usb_product="%s"' % (name, product_name))
     if ((chip == "rp2350") or (chip == "rp2350-riscv")) and (name != "generic_rp2350"):
         print("%s.build.psram_length=0x%d00000" % (name, psramsize))
+    if chip == "rp2350":
+        print("%s.build.mcu=cortex-m33" % (name))
```

Comment:
It's a minor thing, but we do support RISCV mode for everything here (even WiFi and BT).  Is it possible to set the `build.mcu` as part of the `BuildArch` function for the RP2350 instead of hardcoding it here as an ARM?  The R5 mcu should be `rv32imac` if I understand it correctly...

Reply:
Changes made - and verified they work using our rp2350 board in ARM "mode".

Reply:
Awesome, thanks!